docs/examples/cookbook/loading-spinner.md

```diff
@@ -96,7 +96,7 @@ local function Spinner(
 The `CurrentTime` is used to drive the rotation of the loading spinner.
 
 ```Lua
-		Rotation = scope:Computed(function(use)
+		Rotation = scope:Computed(function(use, scope)
 			return (use(props.CurrentTime) * SPIN_DEGREES_PER_SECOND) % 360
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
		Rotation = scope:Computed(function(use)
```

Reply:
reverted